ClemBot.Api/ClemBot.Api.Core/Program.cs

```diff
@@ -166,18 +167,14 @@
     );
 });
 
-builder.Services.AddQuartzServer(options => {
-    options.WaitForJobsToComplete = true;
-});
+builder.Services.AddQuartzServer(options => options.WaitForJobsToComplete = true);
 
-builder.Services.AddCors(options => {
-    options.AddDefaultPolicy(
+builder.Services.AddCors(options => options.AddDefaultPolicy(
         policyBuilder => {
             policyBuilder.AllowAnyOrigin();
             policyBuilder.AllowAnyHeader();
             policyBuilder.AllowAnyMethod();
-        });
-});
+        }));
```

Comment:
the indentation here feels off
personally i would suggest
```suggestion
        }
));
```

Reply:
ultimately the formatter decides, its like black, easier to just roll with it then fight it